src/app/expansion-panel/expansion-panel.component.ts

```diff
@@ -8,12 +9,23 @@ import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angul
   templateUrl: './expansion-panel.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ExpansionPanelComponent {
+export class ExpansionPanelComponent implements OnInit {
+  @Input() questionPanel?: boolean = false;
   @Output() expandedChanged = new EventEmitter<void>();
-  public expanded: boolean = true;
+  public expanded: boolean = defaultConfig.startsExpanded;
+  public buttonStyles: string = defaultConfig.buttonStyles;
+  public contentContainerStyles: string = defaultConfig.contentContainerStyles;
 
   toggle() {
     this.expanded = !this.expanded;
     this.expandedChanged.emit();
   }
+
+  ngOnInit(): void {
+    if (this.questionPanel) {
+      this.expanded = questionPanelConfig.startsExpanded;
```

Comment:
```suggestion
      this.config = {
        ...this.config,
        ...this.configOverride,
      };
```
This should give a config with only the requested parts changed, without having to update if we add more config options.

Reply:
Done :)